src/functions/message/createPoll.js

```diff
@@ -0,0 +1,49 @@
+const { PollLayoutType } = require("discord.js");
+const { Time } = require("../../utils/helpers/customParser.js");
+
+module.exports = async (d) => {
+    const data = d.util.aoiFunc(d);
+    if (data.err) return d.error(data.err);
+
+    const [channelId = d.channel?.id, question, duration, allowMultiselect = "false", ...answers] = data.inside.splits;
+
```

Comment:
add support for json too 
with { name: name,  value:value } for ig either 1 in that we use array or u can do for each ;

Reply:
what i meant was for answers not for everything